credentials/take_browser_console_cred.py

```
@@ -0,0 +1,97 @@
+        print("here: vsphere_vm_name: ", vsphere_vm_name)
+        res_data = acquire_mks_ticket(vsphere_vm_name)
+        print("here: acquire_mks_ticket: res_data: ", res_data)
+
+        if res_data and len(res_data) > 0 and res_data["status"] == "success":
+
+            str_ticket = ""
+            if 'ticket' in res_data['data'] and res_data['data']['ticket'] and res_data['data']['ticket'] is not None:
+                str_ticket = res_data['data']['ticket']
+                print("here: ticket found!: ", str_ticket)
+            else:
+                data = f"ticket not found for VM {vm_name}!"
```

Ruff (0.8.2)

Undefined name `vm_name`

(F821)